codersdk/users.go

```diff
@@ -221,6 +221,28 @@ type OIDCAuthMethod struct {
 	IconURL    string `json:"iconUrl"`
 }
 
+type UserParameter struct {
+	Name       string    `json:"name"`
+	Value      string    `json:"value"`
+	LastUsedAt time.Time `json:"last_used_at" format:"date-time"`
```

Comment:
Does it carry any significant information for FE?

Reply:
I considered adding Workspace name, but it would make the code more difficult to refactor if we changed the behavior to use statically defined parameters or used frequency instead of recency.

Reply:
Ok, so it is not used currently. Shall we remove it for now? We can also extend the API, and adjust database query.